tests/components/file_upload/test_init.py

```diff
@@ -90,9 +90,9 @@ async def test_upload_large_file(
             file_upload.TEMP_DIR_NAME + f"-{getrandbits(10):03x}",
         ),
         patch(
-            # Patch one megabyte to 8 bytes to prevent having to use big files in tests
+            # Patch one megabyte to 50 bytes to prevent having to use big files in tests
```

Comment:
This test was failing because the boundary is now set in time and hits this assertion
https://github.com/aio-libs/aiohttp/blob/9ba9a4e531599b9cb2f8cc80effbde40c7eab0bd/aiohttp/multipart.py#L373

Its not possible to read chunks smaller than the boundary, but I don't think anyone would actually do that in production